### PR TITLE
Add Linux Elastic Defend 8.7.0 known issue doc

### PR DESCRIPTION
**Pull request by nicholasberlin**
https://github.com/elastic/security-docs/issues/3157

### CODE REVIEW

File: docs/release-notes/8.7.asciidoc [hunk 1]
```diff
@@ -19,6 +19,8 @@ If you already upgraded your {agents}, and they're are stuck in the `Updating` s
 
 NOTE: This problem can occur when upgrading {agents} from 8.6.2 or 8.7.0 to any other version. Keep this in mind when upgrading {agents} running 8.6.2 or 8.7.0 to newer versions.
 
+* For Linux Elastic Defend v8.7.0 only, there exists a bug when Malware Prevention is enabled which leads to an Elastic Defend crash. Elastic Defend will enter a bad state when mount points are unmounted. Disabling Malware Protection will remediate the crash. This bug is fixed in 8.7.1.
```
[the remark this answers is not in the document]
Comment (nicholasberlin) [reply docs/release-notes/8.7.asciidoc]: I applied your suggestion, except for one minor change:
`state where mount` -> `state when mount`

Sound good?

[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/release-notes/8.7.asciidoc]: There's a little slippage between "malware _protection_" and "malware _prevention_" here (and I think either should be all lowercase). Protection is the overall feature (1 below), and prevention is a sub-option (2 below):

image

@nicholasberlin could you help identify exactly which option is affected by this issue?

[the remark this answers is not in the document]
Comment (nicholasberlin) [reply docs/release-notes/8.7.asciidoc]: Ah, Nastasha already straightened me out. It should be `Malware protection` (the overall feature), `Malware Prevention` was my mistake.

Both places have `Malware` capitalized, should I lowercase it?

[the remark this answers is not in the document]
Comment (nicholasberlin) [reply docs/release-notes/8.7.asciidoc]: Done

**Review by nfritts — APPROVED** [review]

File: docs/release-notes/8.7.asciidoc [hunk 2]
```diff
@@ -19,6 +19,8 @@ If you already upgraded your {agents}, and they're are stuck in the `Updating` s
 
 NOTE: This problem can occur when upgrading {agents} from 8.6.2 or 8.7.0 to any other version. Keep this in mind when upgrading {agents} running 8.6.2 or 8.7.0 to newer versions.
 
+* Enabling malware protection on Linux hosts might cause {elastic-defend} to enter a failed state when mount points are unmounted, which will stop the integration from working. To fix this, turn off malware protection on the {elastic-defend} integration policy.
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:22]:
```suggestion
* Enabling malware protection on Linux hosts might cause {elastic-defend} to enter a failed state when mount points are unmounted, which will stop the integration from working. To fix this, turn off <<malware protection, malware-protection>> on the {elastic-defend} integration policy.
```

File: docs/release-notes/8.7.asciidoc [hunk 3]
```diff
@@ -19,6 +19,8 @@ If you already upgraded your {agents}, and they're are stuck in the `Updating` s
 
 NOTE: This problem can occur when upgrading {agents} from 8.6.2 or 8.7.0 to any other version. Keep this in mind when upgrading {agents} running 8.6.2 or 8.7.0 to newer versions.
 
+* Enabling malware protection on Linux hosts might cause {elastic-defend} to enter a failed state when mount points are unmounted, which will stop the integration from working. To fix this, turn off <<malware protection, malware-protection>> on the {elastic-defend} integration policy.
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:22]:
```suggestion
* Enabling malware protection on Linux hosts might cause {elastic-defend} to enter a failed state when mount points are unmounted, which will stop the integration from working. To fix this, turn off <<malware-protection,malware protection>> on the {elastic-defend} integration policy.
```

**Pull request merged**